CONTRIBUTING.md

```diff
@@ -85,28 +84,27 @@ Follow these steps to set up your local development environment on Unix/Mac OS:
 
 </details>
 
-
 ## Naming Patterns
 
-### Branch name
+### Branch Name
 ```
 (feat|fix|docs|style|refactor|test|revert)/taskId_task-short-description
 Ex.: feat/1_add-commit-and-branch-name-styles-to-readme
 ```
 
-### Commit name
+### Commit Name
 ```
 (feat|fix|docs|style|refactor|test|revert): update descriptions
-Ex.: bug: update readme
+Ex.: fix: update readme
 ```
 
-### Commit description
+### Commit Description
 ```
 tickets: #taskId, #taskId
 Ex.: tickets: #1, #2, #3
 ```
 
-### Pull-request name
+### Pull Request Name
```

Comment:
This is a good moment to review the naming patterns introduced in b5c5ee8:
- With your present experience, do all pattern still make sense?
- Do the patterns align with our role models, the big open source projects?

Reply:
## Reviewing Existing Naming Patterns

### Branch Name
After reviewing common naming patterns and consulting ChatGPT, I've concluded the following:

Current Pattern: 
```
(feat|fix|docs|style|refactor|test|revert)/taskId_task-short-description
```
This is aligned with common practices. I usually use this pattern, but there's a preference to create branches by dropping the `(feat|fix|docs|style|refactor|test|revert)/taskId_` part.

**Options:**
- [ ] Leave as it is
- [ ] Drop prefix and taskID

### Commit Message
This section should combine the **Commit Name** and **Commit Description** sections, aligning with [issue#25](https://github.com/macxred/cashctrl_api/issues/25).

**Proposed Format:**
```
<Summarize change(s) in around 50 characters or less>

<More detailed explanatory description of the change wrapped into about 72 characters>
```
**Example:**
```
feat: add user authentication

Implement OAuth2 authentication to enhance security and user convenience. Closes #123.
```

### Pull Request Name
Current Pattern:
```
(Feat|Fix|Docs|Style|Refactor|Test|Revert): #taskId description
```
This aligns with common practices. Sometimes we use this pattern, but often we just mention a short description that the PR covers.

**Options:**
- [ ] Leave as it is
- [ ] Use the following pattern:
```
<Summarize Pull Request purpose in around 50 characters or less>
```
**Example:**
```
Add user authentication
```